cmd/vulcan-nuclei/main.go

```diff
@@ -134,16 +134,10 @@ func processNucleiFindings(target string, nucleiFindings []ResultEvent) []*repor
 	vulnerabilities := []*report.Vulnerability{}
 	rv := make(map[string]*report.Vulnerability)
 	for _, v := range nucleiFindings {
-		// Create resources table row.
-		// Avoid store redundant information in the resources table.
-		matched := v.Matched
-		if matched == target {
-			matched = ""
-		}
 		findingRow := map[string]string{
 			"Template":    v.TemplateID,
 			"MatcherName": v.MatcherName,
-			"Matched":     matched,
+			"Matched":     v.Matched,
```

Comment:
What's the reason to change this?

Reply:
The reason is that `matched` was empty when the target string was the same as the matched string. This never would happen with hostname assets because the matched string contains the port and the target does not. However, this would happen with web address assets, for instance in the case where the target is `https://example.com/` and the issue is found in the `/` path. In this case, I see no reason to leave the column empty whereas all the others will contain the full URL where the issue was found (e.g. `https://example.com/vulnerable/`). If we do that, we are leading the users to believe that the table is incomplete instead of clearly telling them in which path the issue was found.

Reply:
Gotcha, very good point.